tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/txprune/HBaseTransactionPruningPlugin.java

```diff
@@ -147,6 +147,7 @@ public long fetchPruneUpperBound(long time, long inactiveTransactionBound) throw
 
     // Get all the current transactional regions
     SortedSet<byte[]> transactionalRegions = getTransactionalRegions();
+    // TODO: Ignore empty
```

Comment:
That TODO was meant for ignoring empty regions from the current transactional regions. I have removed that misleading TODO as part of the last commit.